test/specs/commonTests/isConformant.tsx

```diff
@@ -11,6 +11,14 @@ import helpers from './commonHelpers'
 import * as stardust from 'src/'
 import { felaRenderer } from 'src/lib'
 
+type ShorthandTestOptions = {
+  mapsStringValueToProperty?: string
```

Comment:
grammar nit: 1) In the React community, `props` is not expanded to `property`.  Suggest something like `propKey` or `propName` instead.